changelog/8734-add-date-selector

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: add
+Comment: Behind feature flag - Add a preset date range selector with basic UI to the header of payment activity widget
```

Comment:
This looks out of date – not a big deal but worth making this accurate. 

IMO this PR allows merchants to change the time period used for stats in the payment activity widget. (the date range selector is a "how" not what or why)

Reply:
Also might make a clearer title for the PR!

Reply:
I missed updating this. Thanks for spotting. Updated the changelog here - [d42d16b](https://github.com/Automattic/woocommerce-payments/pull/8927/commits/d42d16b629f555bffe691ba3392e9dd316aa45ce)